components/brave_wallet/common/features.cc

```diff
@@ -19,12 +19,7 @@ const base::FeatureParam<bool> kShowToolbarTxStatus{
 
 BASE_FEATURE(kBraveWalletFilecoinFeature,
              "BraveWalletFilecoin",
-#if BUILDFLAG(IS_ANDROID)
-             base::FEATURE_DISABLED_BY_DEFAULT
-#else
-             base::FEATURE_ENABLED_BY_DEFAULT
-#endif
-);
+             base::FEATURE_ENABLED_BY_DEFAULT);
```

Comment:
We might need a security review for this. cc: @SergeyZhukovsky

Reply:
@kdenhartog wdyt? I personally think we are ok as all of that has been reviewed during desktop implementation and we just added UI part on Android and the UI where we show addresses and etc just follows the existing flow for SOL and ETH.

Reply:
Yeah, let's get a security review open for this, but it should be quick to get approved. Main thing we'd want to do is just double check the UI isn't presenting any odd security bugs which I should be able to double check Monday my time and have final comments ready by the time people are back next week.

Reply:
Filed new security issue: https://github.com/brave/security/issues/1256